db/test/versions/0015_test.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash');
+const helper = require('../helper');
+const testing = require('taskcluster-lib-testing');
+const assert = require('assert').strict;
+const slugid = require('slugid');
+
+suite(testing.suiteName(), function() {
+  const THIS_VERSION = 15;
```

Comment:
` const THIS_VERSION = parseInt(/.*\/0*(\d+)_test\.js/.exec(__filename)[1]);`

Reply:
I'll rebase, now that a 15 has landed :)